src/index.ts

```diff
@@ -1,49 +1,55 @@
+import type { Server as HttpServer } from 'http'
 
+import { serve } from '@hono/node-server'
+import { serveStatic } from '@hono/node-server/serve-static'
+import { Hono } from 'hono'
+import { appendTrailingSlash } from 'hono/trailing-slash'
+import { compress } from 'hono/compress'
+import { secureHeaders } from 'hono/secure-headers'
 
-import { CronJob } from 'cron';
-import express from 'express';
-import { existsSync, mkdirSync } from 'fs';
-import { basepath, server_port } from '../secrets/consts.js';
-import { router as apiRouter } from './api/index.js';
-import { dataDirectory, getResourceURL } from './consts';
-import { collect } from './member-collector/collector';
-import { router as memberRouter} from './member-collector/router';
-import { router as frontendRouter } from './router';
-import path from 'path'
-// Refresh profile images every day
-new CronJob({
-    cronTime: '*/15 * * * *',
-    start: true,
-    timeZone: 'America/Los_Angeles',
-    runOnInit: true,
-    onTick: () => collect()
-})
+import { renderer } from '~lib/renderer'
 
+import admin_router from '~routes/admin'
+import api_router from '~routes/api'
+import account_router from '~routes/auth'
 
-if (!existsSync(dataDirectory)) {
-    mkdirSync(dataDirectory);
-}
+import { requireAdminLogin, requireReadLogin } from '~lib/auth'
+import logger from '~lib/logger'
+import { startWS } from '~lib/sockets'
+import { scheduleTasks } from '~tasks'
+import config from '~config'
+import { startSlack } from '~slack'
 
-// Init Express App
-const app = express()
+const app = new Hono()
 
-app.use("/api", apiRouter)
-app.use("/api", memberRouter)
-app.use("/", frontendRouter)
+// Don't interfere with socket.io
+app.use('/ws/*', async (c, next) => {})
```

Comment:
To make eslint happy
```suggestion
app.use('/ws/*', async (_c, _next) => {})
```